fdbserver/fdbserver.actor.cpp

```diff
@@ -1659,10 +1659,6 @@ struct CLIOptions {
 			case TLSConfig::OPT_TLS_VERIFY_PEERS:
 				tlsConfig.addVerifyPeers(args.OptionArg());
 				break;
-			case OPT_KMS_CONN_DISCOVERY_URL_FILE: {
```

Comment:
The current implementation of FDB native encryption support needs following:
1. KMS URL discovery : Current implementation supports cluster configuration `foundationdb.conf` to supply parameters such as: mode ("file" is the only supported mode at the moment, but in future plans are to extend it to enable integrating sel-serve discoveryService endpoints), local filesystem filepath to enable reading external KMS endpoint details.
2. Validation tokens to enable KMS to authorize a given request for a FDB cluster host. 

Hence, below configurations are valid and few some of them can be asserted upon (as no alternate implementation exists at the moment), but soon enhancements shall be done). Few configs are:
```

	std::string REST_KMS_CONNECTOR_KMS_DISCOVERY_URL_MODE;
	std::string REST_KMS_CONNECTOR_DISCOVER_KMS_URL_FILE;
	std::string REST_KMS_CONNECTOR_VALIDATION_TOKEN_MODE;
	std::string REST_KMS_CONNECTOR_VALIDATION_TOKEN_DETAILS;
	int REST_KMS_CONNECTOR_VALIDATION_TOKEN_MAX_SIZE;
	int REST_KMS_CONNECTOR_VALIDATION_TOKENS_MAX_PAYLOAD_SIZE;
	bool REST_KMS_CONNECTOR_REFRESH_KMS_URLS;
	double REST_KMS_CONNECTOR_REFRESH_KMS_URLS_INTERVAL_SEC;
	std::string REST_KMS_CONNECTOR_GET_ENCRYPTION_KEYS_ENDPOINT;
```

Thanks